Apps/Admin/Tests/Functional/cypress/integration/ui/dashboard.cy.js

```diff
@@ -1,15 +1,36 @@
+function getLastLoginDate(days) {
```

Comment:
This function name is a little confusing - I'd suggest something like `getPastDate` with the parameter renamed to `daysAgo`.

Reply:
Changed.